### PR TITLE
[CDAP-6672]: fix documentation issues for SSHAction

### PR DESCRIPTION
**Pull request by thegreenfrog**
Fix docs issues outlined in https://issues.cask.co/browse/CDAP-6672

### CODE REVIEW

**Comment by knazir** [comment]
This will get overwritten by https://github.com/caskdata/hydrator-plugins/pull/348 since some of the properties have changed as well as their descriptions.

**Comment by knazir** [comment]
@thegreenfrog #348 was merged, you may need to make changes to resolve merge conflicts with the updated descriptions.

File: core-plugins/docs/SSHAction-action.md [hunk 1]
```diff
@@ -29,9 +29,13 @@ pull the private key from the secure key management store in CDAP such as ${secu
 **command:** The command to be executed on the remote host. This includes the filepath of the script and any arguments
 needing to be passed.
 
+<<<<<<< d31ba0836377205c7cbdd3ce449313d0cadf0429
```
Comment (anwar6953) [inline core-plugins/docs/SSHAction-action.md]: this is a merge conflict that needs addressing

**Comment by awholegunch** [comment]
LGTM: Please check my last command make sure it's ok.

**Pull request merged**